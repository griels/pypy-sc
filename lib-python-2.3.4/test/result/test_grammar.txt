command: /usr/bin/python /home/arigo/pypysrc/pypy/tool/alarm.py 900 /home/arigo/pypysrc/pypy/interpreter/py.py --oldstyle /home/arigo/pypysrc/lib-python-2.3.4/test/test_grammar.py
run by: arigo@tismerysoft.de
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 01:16:31 2005
pypy-revision: 11518
OUTPUT TEST
see output in: /home/arigo/pypysrc/lib-python-2.3.4/test/result/test_grammar.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
/home/arigo/pypysrc/lib-python-2.3.4/test/test_grammar.py:42: FutureWarning: hex/oct constants > sys.maxint will return positive values in Python 2.4 and up
  if -2147483647-1 != 020000000000: raise TestFailed, 'max negative int'
/home/arigo/pypysrc/lib-python-2.3.4/test/test_grammar.py:44: FutureWarning: hex/oct constants > sys.maxint will return positive values in Python 2.4 and up
  if 037777777777 != -1: raise TestFailed, 'oct -1'
/home/arigo/pypysrc/lib-python-2.3.4/test/test_grammar.py:45: FutureWarning: hex/oct constants > sys.maxint will return positive values in Python 2.4 and up
  if 0xffffffff != -1: raise TestFailed, 'hex -1'
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
Traceback (application-level):
  File "/home/arigo/pypysrc/lib-python-2.3.4/test/test_grammar.py", line 453 in ?
f()
  File "/home/arigo/pypysrc/lib-python-2.3.4/test/test_grammar.py", line 446 in f
        exec r"""if 1:
  File "<string>", line 2 in ?
TypeError: exec: arg 1 must be a string, file, or code object
========================== closed ==========================
execution time: 9.57444095612 seconds
exit status: 1
