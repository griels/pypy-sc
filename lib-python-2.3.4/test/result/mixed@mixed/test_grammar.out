test_grammar
1. Parser
1.1 Tokens
1.1.1 Backslashes
1.1.2 Numeric literals
1.1.2.1 Plain integers
1.1.2.2 Long integers
1.1.2.3 Floating point
1.1.3 String literals
1.2 Grammar
single_input
file_input
expr_input
eval_input
funcdef
lambdef
simple_stmt
expr_stmt
print_stmt
1 2 3
1 2 3
1 1 1
extended print_stmt
1 2 3
1 2 3
1 1 1
hello world
del_stmt
pass_stmt
flow_stmt
break_stmt
continue_stmt
continue + try/except ok
continue + try/finally ok
testing continue and break in try/except in loop
return_stmt
raise_stmt
import_stmt
global_stmt
exec_stmt
